docker-compose.override.yml

```diff
@@ -111,58 +105,72 @@ services:
-    - registrationApi_apikey=${registrationApi_apikey}
-    - registrationApi_clientSSLCertificateKeystoreLocation=conf/dummy-client-SSL-cert.p12
-    - registrationApi_clientSSLCertificateKeystorePassword=password
-    - registrationApi_harvesterUsername=test_admin
-    - registrationApi_harvesterPassword=password
-    - registrationApi_ipKeyPassword=${registrationApi_ipKeyPassword}
-    - registrationApi_ipStorePassword=${registrationApi_ipStorePassword}
-    - LOG_LEVEL=${LOG_LEVEL}
+      - JAVA_OPTS= -ea -Djava.security.egd=file:/dev/./urandom -Xmx256M -Dspring.profiles.active=docker
+      - FDK_ES_CLUSTERNODES=elasticsearch5:9300
+      - FDK_ES_CLUSTERNAME=elasticsearch
+      - registrationAPI_clusterName=elasticsearch
+      - registrationApi_altinnServiceUrl=http://registration-auth:8080/
+      - registrationApi_apikey=${registrationApi_apikey}
+      - registrationApi_clientSSLCertificateKeystoreLocation=conf/dummy-client-SSL-cert.p12
+      - registrationApi_clientSSLCertificateKeystorePassword=password
+      - registrationApi_harvesterUsername=test_admin
+      - registrationApi_harvesterPassword=password
+      - registrationApi_ipKeyPassword=${registrationApi_ipKeyPassword}
+      - registrationApi_ipStorePassword=${registrationApi_ipStorePassword}
+      - LOG_LEVEL=${LOG_LEVEL}
 
   registration-react:
     build: ./applications/registration-react
     restart: always
     command: npm start
     ports:
-    - "4300:4300"
+      - "4300:4300"
     volumes:
-    - ./applications/registration-react/src:/usr/src/app/src:rw
+      - ./applications/registration-react/src:/usr/src/app/src:rw
     environment:
-    - REGISTRATION_LANGUAGE=${REGISTRATION_LANGUAGE}
-    - NODE_ENV=${NODE_ENV}
+      - REGISTRATION_LANGUAGE=${REGISTRATION_LANGUAGE}
+      - NODE_ENV=${NODE_ENV}
 
   registration-auth:
     restart: always
     ports:
-    - "8077:8080"
+      - "8077:8080"
     environment:
-    - JAVA_OPTS= -ea -Djava.security.egd=file:/dev/./urandom -Xmx256M -Dspring.profiles.active=docker
+      - JAVA_OPTS= -ea -Djava.security.egd=file:/dev/./urandom -Xmx256M -Dspring.profiles.active=docker
 
   reference-data:
     restart: always
     ports:
-    - "8100:8080"
+      - "8100:8080"
     environment:
-    - JAVA_OPTS=-Djava.security.egd=file:/dev/./urandom -Xmx256M -Dspring.profiles.active=docker
+      - JAVA_OPTS=-Djava.security.egd=file:/dev/./urandom -Xmx256M -Dspring.profiles.active=docker
 
   e2e:
     build: ./applications/e2e
     volumes:
-    - ./applications/e2e/test:/usr/src/app/test:rw
-    - ./applications/e2e/reports:/usr/src/app/reports:rw
+      - ./applications/e2e/test:/usr/src/app/test:rw
+      - ./applications/e2e/reports:/usr/src/app/reports:rw
+
+volumes:
+  fuseki-data:
+    driver: local
+    driver_opts:
+      type: none
+      device: ${PWD}/data/fuseki
+      o: bind
+  elasticsearch-data:
+    driver: local
+    driver_opts:
+      type: none
+      device: ${PWD}/data/esdata5
```

Comment:
Denne virker ikke med Windows. Holder på å se etter andre løsninger.

Reply:
er det på grunn at windows har ikke PWD, eller er det noe med samba og rettigheter?

Reply:
Lang historie, men PWD fungerer ikke (egentlig: eksisterer ikke) under vanlig Windows 10 command shell, og kan ikke brukes under GIT BASH på windows da den feil-transformerer stien. Så vi må falle tilbake til en environment variabel.

Reply:
Ok, siden docker-compose.override.yml er maskinspesifikk og docker-compose.yml er generisk (e.g. gjelder også is sky) da er det ok løsning å ha ikke-navnet volumer i maskin og navnet volum i generell. PR oppdatert